### ISSUE TITLE
/api/v1/ gives "action notFound was not found" error

### ISSUE THREAD

**Issue by sanderha** [silverstripe/silverstripe-restfulserver#78]
Accessing the root `/api/v1`gives this error in dev mode:

```
Action 'notFound' isn't available on class SilverStripe\RestfulServer\RestfulServer.
```
Havent looked into the issue yet.

Using recipe-cms ^4.3, php 7.3 and ^2.1 of this module

**Comment by robbieaverill** [comment]
Hi @sanderha, I can't reproduce this issue. I'm using recipe-cms 4.4.1, php 7.1 and 2.1.2.

You can see that `RestfulServer.allowed_actions` defines `notFound` as an allowed action.

Test steps:

* `composer create-project silverstripe/recipe-cms ./restfulserver78 4.4.1`
* `cd restfulserver78 && composer require silverstripe/restfulserver`
* Make environment file and dev/build
* Set `private static $api_access = true` in `Page`
* Flush
* Visit `/api/v1` - returns "Page not found"
* Visit `/api/v1/Page` - returns Page information

Are you able to provide more information to help?

**Comment by sanderha** [comment]
Hmm I can see that my CMS has not generated the 404 page for some reason.. Maybe thats why I'm seting the `Action 'notFound' isn't available` message, because the response is a 404 correctly. 
And you probably did have the 404 error page in the CMS, so thats why you got that displayed when vising `/api/v1`.

I will try to debug some more to see whats wrong

**Comment by robbieaverill** [comment]
Yeah that could be it. The core indicates that it expects to handle its own 404 response, so perhaps the bug is that it’s not doing so
